app/views/categories/_child_categories.html.erb

```diff
@@ -1,4 +1,4 @@
-<% contents.each_with_index do |child_category, i| %>
+<% contents.reject{|c| c.object.type == 'guide' }.each_with_index do |child_category, i| %>
```

Comment:
A couple things. You have this retrieval logic that has leaked out into the view. If it's meaningful it should live in the domain. Secondly - I wonder if `c.guide?` is a thing that encapsulates this check, and if not, could it be? It would let you golf the down to `reject(&:guide?)`